### ISSUE TITLE
Island Disband Causes Player to Drop into Void and not be able to use commands.

### ISSUE THREAD

**Issue by Skizzles** [BG-Software-LLC/SuperiorSkyblock2#193]
**Minecraft's Version**
1.16.5 Yatopia/Paper

**Plugin's Version**: -
SuperiorSkyblock-b107

As shown below on Yatopia the disband causes the player to fall into the void and not send the user to spawn.
Production Environment Test:
https://screenshots.skizzles.me/wAnA3/tAFUBIQi49.mp4

Dev Server Test:

(Yatopia - Build 42)
https://screenshots.skizzles.me/wAnA3/wOZITisA15.mp4

(Paper- Build 573)
https://screenshots.skizzles.me/wAnA3/COjasifI68.mp4

Some possibly related threads that I have seen:
https://github.com/PaperMC/Paper/issues/4342
https://github.com/PaperMC/Paper/issues/3114
https://github.com/Spottedleaf/Tuinity/issues/78

Test with only SSB (b107) on Yatopia:
https://screenshots.skizzles.me/wAnA3/TECoBipa74.mp4

Test with only SSB (b106) on Yatopia:
https://screenshots.skizzles.me/wAnA3/xageDulo24.mp4

In conclusion:
This occurs on B107 with Yatopia installed, versions before 107 do not have the issue with Yatopia. 107 does work with Paper but not Yatopia. Issue seems related to removing entities possibly ASync based off old issues with Paper/Tuinity.

**Comment by Skizzles** [comment]
Looking into this further it seems to be upstream to Purpur and even upstream to Tuinity. Seems like an issue with Tuinity.

**Comment by banicola** [comment]
In fact, we use Tuinity and we have the same problem

**Comment by Skizzles** [comment]
@banicola what Tuinity build do you use?

**Comment by banicola** [comment]
I use the latest Build #218

**Comment by Skizzles** [comment]
Tested as far back in Tuinity as Build # 189 so this seems to be a weird bug in forks that occurs.

@OmerBenGera Not sure what you can do to resolve this issue but this is as much details as I could gather.

**Comment by OmerBenGera** [comment]
@Skizzles Does it happen with older dev builds of SSB as well (before the entities removal)?

**Comment by OmerBenGera** [comment]
Fixed in latest dev build, sorry about that ^^
